roles/katello_repositories/tasks/main.yml

```diff
@@ -2,6 +2,10 @@
 - name: "Set up {{ katello_repositories_environment }} repositories"
   include_tasks: "{{ katello_repositories_environment }}_repos.yml"
 
-- name: enable powertools for libdb_cxx used by qpid-cpp-server-linearstore
+- name: enable codeready-builder for libdb_cxx used by qpid-cpp-server-linearstore
```

Comment:
```suggestion
- name: enable powertools for libdb_cxx used by qpid-cpp-server-linearstore
```

:)

Reply:
so it should be powertools and not codeready-builder?

Reply:
well, for non-Red Hat (so CentOS, Alma, Rocky etc), it's powertools

for Red Hat it's codeready, as in line 9 and further :)

Reply:
ohhh I see...